packages/ordered-list/src/ListItem/ListItem.types.ts

```diff
@@ -0,0 +1,7 @@
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+export type ListItemProps = HTMLElementProps<'li'> & {
+  title?: React.ReactNode;
+  description?: React.ReactNode;
+  darkMode?: boolean;
```

Comment:
You could extend `DarkModeProps` from lib

Reply:
Done